ts/components/conversation/AddMentions.tsx

```diff
@@ -31,10 +31,14 @@ class Mention extends React.Component<MentionProps, MentionState> {
   }
 
   public componentWillMount() {
-    const found = this.findMember(this.props.text.slice(1));
-    this.setState({ found });
 
-    this.tryRenameMention();
+    this.setState({ found: false });
+
+    this.findMember(this.props.text.slice(1)).then((found) => {
+      this.setState({ found });
+      this.tryRenameMention();
```

Comment:
await here or not?

Reply:
I don't think it is necessary, since we don't need the result immediately, we just want it to be executed in the background.